lib/matplotlib/tests/test_axes.py

```diff
@@ -2383,6 +2384,14 @@ def test_scatter_color(self):
         with pytest.raises(ValueError):
             plt.scatter([1, 2, 3], [1, 2, 3], color=[1, 2, 3])
 
+    def test_scatter_color_warning(caplog):
+        # Warn for cases where 'cmap', 'norm', 'vmin', 'vmax'
```

Comment:
Can you parameterize this to test all of the keyword arguments (not just `cmap`)?